openlibrary/templates/covers/add.html

```diff
@@ -25,6 +25,35 @@
 </div>
 
 <script type="text/javascript">
+  Filevalidation = () =>{
```

Comment:
@Aj-KamaL thanks for this. It looks promising.

However I've been working hard to remove JavaScript from inline templates as it's proving super difficult to maintain. In addition to that it can't be tested or linted for errors.

Please move this JavaScript and wire up into openlibrary/plugins/openlibrary/js/  and I can take another pass at review.

Reply:
Also don't forget that all user visible text has to be internationalized. I'm not sure if the OpenLibrary Javascript has an i18n mechanism yet, but, if it doesn't, that may restrict you to HTML based strings.

Reply:
I have moved the JS function to openlibrary/plugins/openlibrary/js/FileValidation.js.
@jdlrobson Can you help me with how to connect FileValidation.js to add.html so that the function Filevalidation inside can be called without using absolute path?

Reply:
That sounds good @Aj-KamaL 

So to include it you'll need to reference that file in openlibrary/plugins/openlibrary/js/index.js

Ideally we'll load the JS only when it's needed, but let's not worry about that right now.

Please push the changes once you've got them loaded on the page and we can tweak it further.

Reply:
Currently the wire up happens here
```
<input type="file" name="file" id="coverFile" value="" onchange="Filevalidation()"/>
```
You can replace this with

```
import onChangeFileValidation from './FileValidation.js';
$('#addcover-form input[type="file"]').on( 'change', onChangeFileValidation );
```